aldryn_events/migrations/0007_set_default_namespaces.py

```diff
@@ -39,11 +41,9 @@ def create_default_namespaces(apps, schema_editor):
                 model_objects = list(model.objects.filter(
                     app_config__isnull=True))
         except (ProgrammingError, OperationalError):
-            new_model = django_apps.get_model(
-                'aldryn_events.{0}'.format(model.__name__))
-            with transaction.atomic():
-                model_objects = new_model.objects.filter(
-                    app_config__isnull=True)
+            # If the above didn't work, do NOT attempt to access django.apps
+            # directly as this will cause issues in future releases. Just pass.
+            pass
```

Comment:
if the above assignment of ``model_objects`` fails with an exception, it variable will be undefined when it is accessed below.